metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -1300,10 +1306,13 @@ def _process_view(
             sql_config=sql_config,
         )
 
+    def get_table_container_key(self, db_name: str, schema: str) -> PlatformKey:
```

Comment:
Shouldn't this be schema_container_key?

Reply:
Its actually a table's container key. It can be of type database or schema. so I didn't use the name schema_container key. But yes, table_conatiner_key has the effect of creating container key of table type. I will change the name to get_parent_container_key.